### ISSUE TITLE
Maintenance: move out Deps.ts

### ISSUE THREAD

**Issue by mrseanryan** [pzavolinsky/ts-unused-exports#65]
The file `Deps.ts` is not an active part of the code base.

For clarity and also to up our code coverage, it could be good to move it out.

options:
1. move it to a tools folder at root
2. move it out to a branch like `crafting/dependencies`

**Comment by mrseanryan** [comment]
Personally I prefer option 2 😬

**Comment by pzavolinsky** [comment]
> Personally I prefer option 2
:+1: 

Remember to add crafting to `.npmignore` so that we don't ship it anymore (at least until we want to include it as an official tool, maybe with a command line option in the main executable).

-----

Since you mention code coverage (maybe a separate PR), here's my config for other projects:

`package.json`
```json
{
  "devDependencies": {
    "codecov": "<this should be latest>",
    "nyc": "<this should be latest>",
  },
  "scripts": {
    "test:cc": "nyc npm run test:unit"
  }
}
```

`.gitignore`
```
.nyc_output
coverage
```

`.nycrc`
```yml
include:
  - src/**/*.ts
  - dist/**/*.js

exclude:
  - src/**/*.d.ts

extension:
  - .ts
  - .js

require:
  - ts-node/register

reporter:
  - lcov
  - text
  - html

sourceMap: true

instrument: true

# Don't show files with full coverage in the reports
skip-full: true

watermarks:
  lines: [80, 100]
  functions: [80, 100]
  branches: [80, 100]
  statements: [80, 100]
```

After running `npm run test:cc`, you can check coverage results in `coverage/index.html`.

**Comment by mrseanryan** [comment]
probably also need to be in `.npmignore`:

```
.nyc_output
coverage
```

**Comment by mrseanryan** [comment]
Hi @pzavolinsky

- made separate issue about code coverage (it links to the above) 
https://github.com/pzavolinsky/ts-unused-exports/issues/66

 just to make sure I have it right:

By option 2 I actually mean delete Deps.ts from master, and make a separate 'crafting' branch that adds Deps.ts back.  
That way Deps is not lost, it can be used later if needed. 

Option 1 is to move Deps to a new folder within master (just to make it clear its not part of main code)

**Comment by pzavolinsky** [comment]
> By option 2 I actually mean delete Deps.ts from master, and make a separate 'crafting' branch that adds Deps.ts back.

Fine by me, go ahead :+1:

**Comment by mrseanryan** [comment]
ok! - the `Deps` removal is included in the Code Coverage PR: https://github.com/pzavolinsky/ts-unused-exports/pull/67

Already have pushed a branch `crafting/dependencies` which is based on that
